src/styles/header.css

```diff
@@ -0,0 +1,70 @@
+/* Styles go here */
+
+body {
+  --hover-color: #00acdc;
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+header {
```

Comment:
Add a classname to the 'header' element and replace the 'header' selector with the classname, e.g. '.header'.